public/pages/EditFeatures/utils/constants.ts

```diff
@@ -33,6 +33,7 @@ export enum SAVE_FEATURE_OPTIONS {
 
 export const AGGREGATION_TYPES = [
   { value: 'avg', text: 'average()' },
+  { value: 'value_count', text: 'count()' },
```

Comment:
small: is there a reason why this is `value_count` and not `count`?

Reply:
Check this https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-valuecount-aggregation.html

Reply:
Makes sense, thanks.